test/make.sh

```diff
@@ -92,8 +90,8 @@ function generate_docs() {
   echo "Generating markdown docs with terraform-docs"
   TMPFILE=$(mktemp)
   #shellcheck disable=2006,2086
-  for j in `for i in $(find . -type f | grep \.tf$) ; do dirname $i ; done | sort -u` ; do
-    terraform-docs markdown "$j" > "$TMPFILE"
+  for j in $(for i in $(find . -type f -path autogen -prune | grep \.tf$); do dirname $i; done | sort -u); do
```

Comment:
This change appears to be unrelated to the branch topic.

Reply:
Fixed.